db/models.py

```diff
@@ -14,3 +14,44 @@ class Actor(models.Model):
 
     def __str__(self) -> str:
         return f"{self.first_name} {self.last_name}"
+
+
+class Movie(models.Model):
+    title = models.CharField(max_length=255)
+    description = models.TextField()
+    actors = models.ManyToManyField(Actor, related_name="movies_actor")
```

Comment:
The `related_name='movies_actor'` is inconsistent with typical naming conventions. Consider using `related_name='movies'` to reflect the relationship from `Actor` to `Movie` more clearly.